frontend/components/issues/js/issues.service.js

```diff
@@ -536,7 +536,7 @@
 			if (issue.viewpoint && issue.viewpoint.hasOwnProperty("hideIfc")) {
 				TreeService.setHideIfc(issue.viewpoint.hideIfc);
 			}
-			TreeService.showAllTreeNodes();
+			TreeService.showAllTreeNodes(false);
```

Comment:
This might be causing a problem in the case where issue has no hidden objects and no selected objects as I believe this always expects something else to update the tree. Perhaps something to add in the else case of checking for groups, handleHighlights, handleHidden, handleTree?

e.g. http://127.0.0.1/charence/1556f6b2-7db9-4ce3-bb73-24b6ad82073c/5fbb96c8-6dbf-4ab1-b07e-57a36cb126f8 click on ISSUE "Architectural.7 ABSOLUTELY EVERYTHING NO SELECTION"

You can see the tree updating, but the viewer doesn't change

Reply:
Is there a quick a check for that?